client/client.go

```diff
@@ -63,18 +63,28 @@ type Config struct {
 	// BaseURL contains the base URL where snappy daemon is expected to be.
 	// It can be empty for a default behavior of talking over a unix socket.
 	BaseURL string
+
+	// DisableAuth controls if the client should try to read the auth.json
```

Comment:
controls whether the client should send an Authorization header from reading the auth.json data.